recipes/laszip/COPYING

```diff
@@ -0,0 +1,511 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
This file is not necessary

Reply:
I have no strong feelings here. It was added per @CJ-Wright request and according to the [docs](https://github.com/conda-forge/staged-recipes/blob/a504af81c05491bf7b0b018b2fa1efe64767985c/recipes/example/meta.yaml#L52-L55). Shall I go ahead and remove?

Reply:
I meant that you need the `license_file` field in the `meta.yaml` so that a license would be associated with the package. If the license file comes from the package itself it is better. You can remove that file (but not the key from the `meta.yaml`).